hyped.code-workspace

```diff
@@ -35,5 +35,8 @@
     "editor.detectIndentation": false,
     "C_Cpp.default.cppStandard": "c++11",
     "C_Cpp.default.cStandard": "c11",
+    "files.associations": {
```

Comment:
This shouldn't be here : ) Other than that looking good

Reply:
Done now :)